source/vibe/http/common.d

```diff
@@ -601,7 +601,7 @@ struct CookieValueMap {
 
 	inout(string)* opBinaryRight(string op)(string name) inout if(op == "in")
 	{
-		foreach(c; m_entries)
+		foreach(ref c; m_entries)
```

Comment:
This shouldn't be needed. I did the same change in #972, but Sonke fixed it in https://github.com/rejectedsoftware/vibe.d/commit/29d3fad3b03b0a65a8d91ad0152d60ad965896d2